src/mtev_thread.c

```diff
@@ -95,6 +96,12 @@ static mtev_boolean mtev_disable_binding = mtev_false;
 uint32_t
 mtev_thread_id(void) {
   if(!mtev_current_thread_id) mtev_current_thread_id = getthreadid();
+  aco_t *co = aco_co();
```

Comment:
Should this use the new `aco_co_thread()` ?

Reply:
yup